mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -404,11 +401,11 @@ To get a list of all modules a collection will load, you can use the
 
 ```
 $ module describe my-collection
-1) example/1.2.3                                        6) imkl/11.3.3.210-iimpi-2016b 
-2) GCCcore/5.4.0                                        7) intel/2016b 
-3) icc/2016.3.210-GCC-5.4.0-2.26                        8) examplelib/1.2-intel-2016b 
-4) ifort/2016.3.210-GCC-5.4.0-2.26                      9) secondexample/2.7-intel-2016b 
-5) impi/5.1.3.181-iccifort-2016.3.210-GCC-5.4.0-2.26
+1) example/1.2.3                                        6) imkl/11.3.3.210-iimpi-2023a 
+2) GCCcore/5.4.0                                        7) intel/2023a 
+3) icc/2023.3.210-GCC-5.4.0-2.26                        8) examplelib/1.2-intel-2023a 
```

Comment:
Same here, wrong `GCCcore`, there should be no `icc/*`, etc.

Reply:
Also fixed in [use real module list in 'module load' and 'module unload' examples](https://github.com/hpcugent/vsc_user_docs/pull/710/commits/a3a953258d30097f742509882a19a3eed1455112)